Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -10,10 +10,12 @@
 #import <AxolotlKit/PreKeyBundle.h>
 #import <Curve25519Kit/Randomness.h>
 #import <SignalMessaging/Environment.h>
+#import <SignalServiceKit/MIMETypeUtil.h>
```

Comment:
There is going to be rebase hell with the bubble collapse work; let's merge this first?